File: oj-28tech/java/Cấu trúc rẽ nhánh & Toán tử/ifelse07.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        // Reading input
        long a = sc.nextLong(),  b = sc.nextLong();
        
        // Find the largest number <= a that is divisible by b
        long largest_divisible = a / b * b;

        // Find the smallest number >= a that is divisible by b
        long smallest_divisible = (a + b - 1) / b * b;
        
        // Outputting results
        System.out.print(largest_divisible + " " + smallest_divisible);

        // Closing scanner (not essential in this case, but good practice)
        sc.close();
    }
}

File: oj-28tech/c++/Cấu trúc rẽ nhánh & Toán tử/ifelse07.cpp
#include <iostream>

using namespace std;

int main() {
    long long a, b;
    cin >> a >> b;

    // Find the largest number <= a that is divisible by b
    long long largest_divisible = (a / b) * b;

    // Find the smallest number >= a that is divisible by b
    long long smallest_divisible = (a + b - 1) / b * b;

    cout << largest_divisible << " " << smallest_divisible << endl;

    return 0;
}
